homework-1/src/templates/stiky.html

```diff
@@ -0,0 +1,5 @@
+<div class="stiky__footer-texts">
+    <a href="#tipsLogo" class="stiky__footer-text">TIPS</a>
+    <a href="#sliderLogo" class="stiky__footer-text header__footer-text-border">GLOSSARY</a>
```

Comment:
`header__footer-text-border` - you do not have css for this class. Remove it.

Find another way to add separator between this links - you can create left border for every link except first. Or you can create pseudo element.